dev-client/ios/Podfile

```diff
@@ -26,6 +27,29 @@ if linkage != nil
 end
 
 target 'Terraso LandPKS' do
+  use_expo_modules!
+  post_integrate do |installer|
+    begin
+      expo_patch_react_imports!(installer)
+    rescue => e
+      Pod::UI.warn e
+    end
+    begin
+      expo_patch_react_imports!(installer)
+    rescue => e
+      Pod::UI.warn e
+    end
+    begin
+      expo_patch_react_imports!(installer)
+    rescue => e
+      Pod::UI.warn e
+    end
+    begin
+      expo_patch_react_imports!(installer)
+    rescue => e
+      Pod::UI.warn e
+    end
```

Comment:
question: does this need to be run four times?

Reply:
probably not? the ios and android file changes were autogenerated by the expo install command, but i had to run it several times with minor changes to get it to work so it might just be a sloppy script that adds the command each time it is run instead of checking if it already exists

Reply:
Agreed. Please remove the duplication (and perhaps file a bug on the installer script).